03-transactions.md

```diff
@@ -449,6 +536,7 @@ These pubkeys are simply generated by addition from their base points:
 	pubkey = basepoint + SHA256(per_commitment_point || basepoint) * G
 
 The `localpubkey` uses the local node's `payment_basepoint`;
+The `remotepubkey` uses the remote node's `payment_basepoint`;
```

Comment:
Why was this added? To make things consistent without applying our new feature bit modifications? 

Just from scanning this PR, I think we may want to take a step back (as a whole) and re-consider how we express feature bit modulations in the spec itself. If we follow with the current style, I can see a future where we have a bunch of strewn pseudo-conditionals (if `option_`), which makes the spec harder to read overtime since the reader must first parse the base functionality (BOLT 1.0), then apply each modification caused by a negotiated option in order to gain a valid working model in their minds.

Reply:
Added this because it was missing. It isn't related to anchor outputs and I will split it out to its own commit eventually. If this change is correct?

Reply:
Agreed that the "pseudo-conditionals" don't really scale to the many versions of commitments, payments, etc that we inevitably seem to be getting.

Rather than trying to make it one consistent document, could it work to do 'releases' in which we only describe the latest state of the spec? It is probably harder to cross check backwards compatibility (need to diff with previous release), but we wouldn't need those conditionals.

Reply:
Closing this thread and leaving discussion about a different spec repo format for another PR